MetaMask/metamask-extension
# Creating story for `account-mismatch-warning`

## Explanation

- Created story for `account-mismatch-warning` component and replaced a deprecated constant
- #18714 
- `ui/components/ui/account-mismatch-warning`



## Screenshots/Screencaps



### Before


#### Story does not exist in `ui/components/ui` folder
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/ed030a9b-fb5f-4831-b897-f544c57d1933)

### After


![image](https://github.com/MetaMask/metamask-extension/assets/79097544/74deefd2-74c6-41f8-ba87-60f73de2af83)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/dd2d7200-198e-45bf-b2c3-98aa7ed41db7)

covered `jest` tests
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/da221398-1823-4a95-af43-f75020f9a9b8)

## Manual Testing Steps

- Pull this branch
- Run `yarn storybook`
- Check `AccountMismatchWarning` story in UI component



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
